### PR TITLE
ICMSLST-2196 Add django.contrib.sites package to support several sites.

### PR DESCRIPTION
**Pull request by MattHolmes123**
Features can now be enabled / disabled depending on the URL.
Importer and Exporter access requests have been used as an example.

Other changes:
  - Add data migration to create sites
  - Add script to set ICMS site urls.

### CODE REVIEW

File: web/sites/__init__.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+            return f(request, *args, **kwargs)
+
+        return _wrapped_view
+
+    return decorator
+
+
+def is_exporter_site(site: Site) -> bool:
+    return site.name == EXPORTER_SITE_NAME
```
Comment (marcuspp) [inline web/sites/__init__.py:53]: The site name can be changed in django admin. Maybe consider using domain and make it uneditable in django admin.

**Review by marcuspp — APPROVED** [review]

File: web/sites/__init__.py (hunk 1 above)
Comment (MattHolmes123) [reply web/sites/__init__.py:53]: Domain is unique, we may want multiple domains to mean the same thing "An Exporter Site".

Comment (MattHolmes123) [reply web/sites/__init__.py:53]: I considered updating the admin interface to check name was one of the 3 supported values in a custom form.

Comment (MattHolmes123) [reply web/sites/__init__.py:53]: https://uktrade.atlassian.net/browse/ICMSLST-2318

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**